response_operations_ui/static/js/disable-send.js

```diff
@@ -1,5 +1,11 @@
-$(document).ready(function() {
-  $("#create-message-form").submit(function() {
-    $("#btn-send-message").prop("disabled", true);
-  });
-});
+(function(window) {
+    const createMessageForm = document.getElementById('create-message-form');
+    const buttonSendMessage = document.getElementById('btn-send-message');
+
+    if (createMessageForm && buttonSendMessage) {
+        createMessageForm.addEventListener('submit', function(event) {
+            buttonSendMessage.attributes.disabled = 'disabled';
+        });
+    }
+}(window));
+
```

Comment:
Sadly, because the script isn't loaded until late in the markup - plan will be to have an actual DOMContentLoaded handler when we move to a bundled execution.